content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -274,19 +274,14 @@ After 15 seconds, view Pod events to verify that the liveness check has not fail
 kubectl describe pod etcd-with-grpc
 ```
 
-Before Kubernetes 1.23, gRPC health probes were often implemented using
-[grpc-health-probe](https://github.com/grpc-ecosystem/grpc-health-probe/),
-as described in the blog post
-[Health checking gRPC servers on Kubernetes](/blog/2018/10/01/health-checking-grpc-servers-on-kubernetes/).
-The built-in gRPC probe's behavior is similar to the one implemented by grpc-health-probe.
-When migrating from grpc-health-probe to built-in probes, remember the following differences:
-
-- Built-in probes run against the pod IP address, unlike grpc-health-probe that often runs against
-  `127.0.0.1`. Be sure to configure your gRPC endpoint to listen on the Pod's IP address.
-- Built-in probes do not support any authentication parameters (like `-tls`).
+When using a gRPC probe, there are some technical details to be aware of:
+
+- The probes run against the pod IP address.
+  Be sure to configure your gRPC endpoint to listen on the Pod's IP address.
```

Comment:
I think a hostname can be used but defaults to the pod's IP, according to the KEP
https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2727-grpc-probe
/sig node

Reply:
/lgtm cancel

Reply:
Double checked. The endpoint has to listen on the Pod IP, but the probe can work using hostname or IP address.
Fixed the text for clarity.